iroh-netcheck/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "iroh-netcheck"
+version = "0.1.0"
```

Comment:
lets use the community version number `0.28` for this, given it has the `iroh-` prefix